dependencies.sh

```diff
@@ -11,8 +11,8 @@ export BYOND_MINOR=1637
 export RUST_G_VERSION=3.1.0
 
 #node version
-export NODE_VERSION=20
-export NODE_VERSION_LTS=20.12.0
+export NODE_VERSION_LTS=20.13.0
+export NODE_VERSION_COMPAT=20.2.0
```

Comment:
add a comment saying you need to make sure this works on windows 7 before changing this, or else we'll end up here again.